src/DependencyInjection/Configuration.php

```diff
@@ -36,6 +36,7 @@ public function getConfigTreeBuilder()
                         'src' => '%kernel.project_dir%/public',
                         'dest' => '.',
                         'fail_if_missing' => true,
+                        'ignore_dot_files' => true,
```

Comment:
Perhaps this should be false to include by default dotfiles from the public dir?
Or let the developper handle it per-case, should be fine.

Reply:
The only web-related public dotfile i can think of is the `.htaccess`.